src/services/posts/unlike-post.test.js

```diff
@@ -5,32 +5,53 @@ const unlikePost = require("./unlike-post")
 const {fixtures} = require('../../../test/utils')
 
 
-describe('UNLIKE a gif', () => {
+describe('UNLIKE a post', () => {
```

Comment:
this is cool, but let's deal with the article and gif tests here in this same file as as inner `describe` blocks. Kindly check out the format here: https://github.com/tbf-challenge/teamwork-be/blob/main/src/services/posts/like-post.test.js

Reply:
@jocrah Done

Reply:
okay, let's kindly take out the `delete-article.test.js` then

Reply:
@jocrah Done